### ISSUE TITLE
broken when npm loglevel is set

### ISSUE THREAD

**Issue by Fandekasp** [mtscout6/syntastic-local-eslint.vim#9]
Try:

```
npm config set loglevel info
```

Then the local eslint will not be found anymore in the list of checkers available

Revert to:

```
npm config set loglevel silent 
```

and eslint checker will be back.

The log infos are probably captured by the plugin when they should be ignored.

**Comment by mtscout6** [comment]
I could see that being a problem. If you've got the time to submit a PR to fix it then I'll review it and bring it in. Or it may be worth it to just add a note to the README about this as a known issue.

**Comment by nightsdark** [comment]
I'm seeing errors now, using the plugin but, I had to set loglevel silent and it started working. I've been missing errors, just some errors where displaying in vim. I tried different settings within eslintrc.js in project root but, eslint gave unusual errors running $eslint app.js from command line. About an hour ago, my computer froze and I rebooted. I still had no errors in vim but, running $eslint app.js from command line showed nicely formatted error output. Till installing this plugin and
$  npm config set loglevel silent.
